pkg/pagerduty/types.go

```diff
@@ -0,0 +1,6 @@
+package pagerduty
+
+type Alert struct {
+	ID   string
+	Body map[string]interface{}
```

Comment:
I am not sure we want to use a map[string]interface{}  I would like to discuss this usage here at standup. I think we need a more defined struct. Even if it means marshalling the data temporarily into map[string]interface{}, then doing type checking to construct our own body with known type

Reply:
This was done to separate the getalerts from #16 , in an ideal world I would save the external struct here and marshal it later

Reply:
added a comment to explain why the body is not a normal struct